src/CarbonSDK.ts

```diff
@@ -144,7 +144,7 @@ class CarbonSDK {
     this.admin = new AdminModule(this);
     this.order = new OrderModule(this);
     this.lp = new LiquidityPoolModule(this);
-    this.vault = new VaultModule(this);
+    this.perpsliquidity = new PerpsLiquidityModule(this);
```

Comment:
Think it's best not to make any changes to the vault module cos it might cause merge conflicts and inconvenience down the road. Wait for everything to finalise first then rename/refactor bah.